src/components/View/Lottery/components/DrawInfo/Detail.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
```

Comment:
Is there a reason for importing React? 👀

Reply:
i forgot this one also :)